circuits/circom/float.circom

```diff
@@ -1,4 +1,5 @@
 pragma circom 2.0.0;
+
 include "../node_modules/circomlib/circuits/bitify.circom";
```

Comment:
But here it seems we import the bitify the old way still?

Reply:
must have missed this import then, good catch